src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -25,14 +25,18 @@ public void initialBeforeYouStart(CaseDetails caseDetails) {
         var acasBinaryUrl = createDocLinkDefault(caseDetails.getCaseId());
 
         var documentCollection = caseDetails.getCaseData().getDocumentCollection();
-        for (DocumentTypeItem d : documentCollection) {
-            if (d.getValue().getTypeOfDocument().equals(ET1_DOC_TYPE)) {
-                et1BinaryUrl = createDocLinkBinary(d);
-            }
-            if (d.getValue().getTypeOfDocument().equals(ACAS_DOC_TYPE)) {
-                acasBinaryUrl = createDocLinkBinary(d);
+        if (documentCollection != null && !documentCollection.isEmpty()) {
+            for (DocumentTypeItem d : documentCollection) {
+                if (d.getValue().getTypeOfDocument().equals(ET1_DOC_TYPE)) {
```

Comment:
It's usually safer to do the comparison the other way round to avoid NPE
`if (ET1_DOC_TYPE.equals(d.getValue().getTypeOfDocument()) {`